docker/php/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install \
         iconv mcrypt mbstring intl pdo pdo_mysql pdo_pgsql
 
+RUN echo "date.timezone=Europe/Paris" > /usr/local/etc/php/conf.d/date_timezone.ini
```

Comment:
I'll have a look how to pass arguments to the build from the compose to the
dockerfile, but I'm not a big fan either of using a version that is only
available since a month. It forces everyone to upgrade (oups I was still
using 1.4 )


On Fri, Feb 26, 2016 at 10:05 AM, Fabien Meurillon <notifications@github.com
> wrote:

> In docker/php/Dockerfile
> <https://github.com/wallabag/wallabag/pull/1727#discussion_r54218638>:
>
> > @@ -5,6 +5,8 @@ RUN apt-get update && apt-get install -y \
> >      && docker-php-ext-install \
> >          iconv mcrypt mbstring intl pdo pdo_mysql pdo_pgsql
> >
> > +RUN echo "date.timezone=Europe/Paris" > /usr/local/etc/php/conf.d/date_timezone.ini
>
> Maybe this could rely on a build argument
> <https://docs.docker.com/engine/reference/builder/#arg>, so it will be
> overridable in docker-compose.yml without change in Dockerfile ?
> (starting with *docker-compose* 1.6)
>
> Is there some non-"french timezoned" developers on Wallabag ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wallabag/wallabag/pull/1727/files#r54218638>.
>